src/data/terraform.json

```diff
@@ -1,6 +1,7 @@
 {
   "slug": "terraform",
   "name": "Terraform",
+  "basePaths": ["language"],
```

Comment:
Does this need all 12 of the docs page paths?

Reply:
You're absolutely right - i forgot i needed to add these since i wasn't testing links yet. Added! ✅